### DISCUSSION TITLE
当设置valueinCols为false的时候，可以让总计根据行维度值来分组并且只展示指定行维度吗

### DISCUSSION THREAD

**Discussion by MuRui666** [antvis/S2#3030] [Q&A]
如下图所示，当设置valueinCols为false的时候，总计行中还显示了数值分组，希望实现只显示城市维度分组
![image](https://github.com/user-attachments/assets/b5b406c6-169f-4c4c-9564-4abfcbed34d4)

**Comment by lijinke666** [comment]
不支持, 文档没写就是不支持
